libraries/botbuilder-azure-blobs/package.json

```diff
@@ -30,16 +30,17 @@
     "@azure/storage-blob": "^12.24.0",
     "botbuilder-core": "4.1.6",
     "botbuilder-stdlib": "4.1.6",
-    "p-map": "^4.0.0",
+    "p-map": "^7.0.3",
     "zod": "^3.23.8",
     "@azure/core-http": "^3.0.4"
   },
   "scripts": {
     "build": "tsc -b",
     "build-docs": "typedoc --theme markdown --entryPoint botbuilder-azure-blobs --excludePrivate --includeDeclarations --ignoreCompilerErrors --module amd --out ..\\..\\doc\\botbuilder-azure-blobs .\\lib\\index.d.ts --hideGenerator --name \"Bot Builder SDK - Azure Blobs\" --readme none",
-    "clean": "rimraf _ts3.4 lib tsconfig.tsbuildinfo",
+    "clean": "rimraf _ts3.4 lib vendors tsconfig.tsbuildinfo",
     "depcheck": "depcheck --config ../../.depcheckrc",
     "lint": "eslint . --config ../../eslint.config.cjs",
+    "prebuild": "tsup ./node_modules/p-map/*.js --format cjs --dts --out-dir vendors/p-map --clean --sourcemap",
```

Comment:
Add the vendors folder to the files property in this package.json like...
![imagen](https://github.com/user-attachments/assets/2aed328f-aef1-481b-8a3b-2a914b04d703)

Reply:
fixed